code/modules/spacepods/spacepods.dm

```diff
@@ -463,6 +497,94 @@
 		to_chat(usr, "You stop entering the pod.")
 	return
 
+/obj/spacepod/proc/attempt_load_cargo(atom/movable/moved, mob/user)
+	if(!ES || !istype(ES))
+		to_chat(user, "<span class='warning'>The pod has no equipment datum, or is the wrong type, yell at pomf.</span>")
+		return
+	if(!ES.cargo_system)
+		to_chat(user, "<span class='warning'>The pod has no cargo system.</span>")
+		return
+	if(locked)
+		to_chat(usr, "<span class = 'warning'>\The [src] is locked.</span>")
+		return
+	if(usr.incapacitated() || usr.lying) //are you cuffed, dying, lying, stunned or other
+		return
+	if (!ishigherbeing(usr))
+		return
+	if(ES.cargo_system.stored)
+		to_chat(user, "<span class='warning'>The pod has no room in its cargo bay.</span>")
+
+	visible_message("<span class='notice'>[usr] starts to load \the [moved] into \the [src].</span>")
+
+	if(do_after(usr, src, 4 SECONDS))
+		if(ES.cargo_system.stored)
+			//Something loaded when you weren't looking!
+			to_chat(user, "<span class='warning'>The pod has no room in its cargo bay.</span>")
+			return
+		moved.forceMove(src)
+		ES.cargo_system.stored = moved
+		src.add_fingerprint(usr)
+		moved.add_fingerprint(usr)
+		to_chat(usr, "<span class = 'notice'>You load \the [moved] into \the [src].</span>")
+	else
+		to_chat(usr, "You stop loading the pod.")
+	return
+
+/obj/spacepod/verb/attempt_unload_cargo()
+	set category = "Spacepod"
+	set name = "Unload Cargo"
+	set src in oview(1)
+
+	if(!ES || !istype(ES))
+		to_chat(usr, "<span class='warning'>The pod has no equipment datum, or is the wrong type, yell at pomf.</span>")
+		return
+	if(!ES.cargo_system)
+		to_chat(usr, "<span class='warning'>The pod has no cargo system.</span>")
+		return
+	if(locked)
+		to_chat(usr, "<span class = 'warning'>\The [src] is locked.</span>")
+		return
+	if(usr.incapacitated() || usr.lying) //are you cuffed, dying, lying, stunned or other
+		return
+	if (!ishigherbeing(usr))
+		return
+	if(!ES.cargo_system.stored)
+		to_chat(usr, "<span class='warning'>The pod has nothing in the cargo bay.</span>")
+		return
```

Comment:
You've used this block twice; time to turn it into a proc :^)
Same comment for line 539 applies as in the last instance of this